mittab/libs/tab_logic.py

```diff
@@ -286,6 +286,7 @@ def validate_round_data(round_to_check):
     If any of these fail we raise a specific error as to the type of error
     """
 
+    # TODO (Kevin) remove this? Can just assign judges later.
```

Comment:
This error is used to update the UI and display a message that there aren't enough judges. Using errors for control flow probably isn't the greatest thing, but that's something to keep in mind before changing this